### PR TITLE
ubuntu 20.04

### PR DESCRIPTION
**Pull request by jr0d**
Adds ubuntu support to kib

### CODE REVIEW

File: images/ami/ubuntu-focal.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  ami_filter_name: "ubuntu/images/hvm-ssd/ubuntu-focal-20.04-amd64-server*"
+  ami_filter_owners: "099720109477"
+  source_ami: ""
+  distribution: "Ubuntu"
+  distribution_version: "20.04"
+  ssh_username: "ubuntu"
+  root_device_name: "/dev/sda1"
+  volume_size: "15"
+
+build_name: "ubuntu-focal"
```
Comment (tillt) [inline images/ami/ubuntu-focal.yaml:12]: can we make it "ubuntu-20" to be in line with the other distros?

Comment (jr0d) [reply images/ami/ubuntu-focal.yaml:12]: ubuntu 20.04 or 20.14?

**Review by tillt — APPROVED** [review]
Thanks and awesome work for sure!

**Comment by tillt** [comment]
Gave this a spin (in combo with https://github.com/mesosphere/konvoy-image-builder/pull/98 - build an image, used the image for provisioning GPU workers of a cluster, installed konvoy2, kommander2 - cluster came up perfectly fine.

**Pull request merged**